### ISSUE TITLE
Fix selected reaction pill color for custom themes.

### ISSUE THREAD

**Issue by th0mcat** [vector-im/element-web#12628]
### Description

For custom themes, the selected reaction pill produces some interesting results.

For example:

![image](https://user-images.githubusercontent.com/37344783/76007189-4495cd80-5ed3-11ea-89c3-5f572c48d7f5.png)

If I edit the following lines in `theme-dark-custom.css`:

```
.mx_ReactionsRowButton.mx_ReactionsRowButton_selected {
    background-color: #1f6954;
    border-color: var(--accent-color)
}
```

to:

```
.mx_ReactionsRowButton.mx_ReactionsRowButton_selected {
    background-color:var(--accent-color);
    border-color: var(--accent-color)
}
```

My  selected reaction pill changes to:

![image](https://user-images.githubusercontent.com/37344783/76007699-149afa00-5ed4-11ea-82bd-413c25f812cb.png)

I think adding `reaction-pill-color` and `reaction-pill-secondary-color` as allowed variables in the main `config.json` would fix the issue.

### Version information

- **Platform**: Web 
- **Browser**: Firefox
- **OS**: Windows
- **URL**: chat.thomcat.rocks, v1.5.11-29-g872a7a74

**Comment by turt2live** [comment]
Needs design mostly to ensure the effect of your change is intended, as I think there's some opacity lost.

**Comment by Oha-you** [comment]
@SimonBrandner and this one should be fixed as well by matrix-org/matrix-react-sdk/pull/7270

Support for a new variable has been added to custom themes: `reaction-row-button-selected-bg-color`
Non-selected reaction uses `roomlist-background-color`, which I think is fine.
